website/docs/externaldata.md

```diff
@@ -468,14 +468,34 @@ To ensure a request to the Gatekeeper webhook is coming from the API server, Gat
       ca.key: <ca-key>
       tls.crt: <tls-crt>
       tls.key: <tls-key>
-      clientca.crt: <apiserver's ca.crt>
+      clientca.crt: <ca-crt>
     metadata:
       ...
       name: <gatekeeper-webhook-service-name>
       namespace: <gatekeeper-namespace>
     type: Opaque
     ```
-3. You will need to make sure the K8s API Server includes appropriate certificate while sending requests to the webhook, otherwise webhook will not accept these requests and will log error of `tls client didn't provide a certificate`. To make sure API server attaches correct certificate to requests being sent to webhook, you must specify the location of the admission control configuration file via the `--admission-control-config-file` flag while starting the API server. Here is an example admission control configuration file:
+3. Generate client certificates signed with CA authority to be attached by API server while talking to Gatekeeper webhook. Gatekeeper webhook expects API server to attach the certificate that has CN name as `kube-apiserver`. If certificate with anyother CN name is used, webhook will throw the error and not accept the request.
+
+    - Generate private key for API server
+
+      ```bash
+      openssl genrsa -out apiserver-client.key 2048
+      ```
+    - Generate a CSR for API server
+    
+      ```bash
+      openssl req -new -key apiserver-client.key -out apiserver-client.csr -subj="/CN=kube-apiserver"
+      ```
+    
+    - Generate public key for API server
+      | Replace `<ca-crt>` and `<ca-key>` with correct values
+
+      ```bash
+      openssl x509 -req -in apiserver-client.csr -CA <ca-crt> -CAkey <ca-key> -CAcreateserial -out apiserver-client.crt -days 825
+      ```
+
+4. You will need to make sure the K8s API Server includes appropriate certificate while sending requests to the webhook, otherwise webhook will not accept these requests and will log error of `tls client didn't provide a certificate`. To make sure API server attaches correct certificate to requests being sent to webhook, you must specify the location of the admission control configuration file via the `--admission-control-config-file` flag while starting the API server. Here is an example admission control configuration file:
```

Comment:
```suggestion
4. You will need to make sure the K8s API Server includes the appropriate certificate while sending requests to the webhook, otherwise webhook will not accept these requests and will log an error of `tls client didn't provide a certificate`. To make sure the API server attaches the correct certificate to requests being sent to webhook, you must specify the location of the admission control configuration file via the `--admission-control-config-file` flag while starting the API server. Here is an example admission control configuration file:
```